tools/development/parser/meson.build

```diff
@@ -57,6 +53,8 @@ gen_grammar = configure_file(input : 'gen_grammar.py.in',
   configuration : bison_cdata)
 
 # Custom targets
+python3 = find_program('python3')
```

Comment:
I think https://mesonbuild.com/Python-module.html#find_install is the more cross-platform compatible way.

Reply:
Updated, thanks.